module3/lessons/Week3/TellingYourStory.md

```diff
@@ -0,0 +1,118 @@
+
+Storytelling is our first step in the job search at Turing. You are a member of the software industry now, and as you start connecting with others in the industry, you need to be able to tell the story of your transition – how did you get here? How do you uniquely make up a part of this industry?
+
+Let’s start by thinking about stories. What do you love about stories? Emma Coats, a former story artist for Pixar, shared her 22 rules for storytelling [here](https://www.washingtonpost.com/blogs/comic-riffs/post/pixar-tips-brave-artist-emma-coats-shares-her-storytelling-wit-and-wisdom-on-twitter%20followher/2012/06/25/gJQADaxd2V_blog.html), and we’ll apply some of them to our storytelling. We’re going to start with rule #10:
+
+*Pull apart the stories you like. What you like in them is a part of you; you’ve got to recognize it before you can use it.*
+
+<section class="call-to-action" markdown="1">
+Think about a story that is meaningful to you and why you connect with it. After time to think on your own, you'll jump into breakout rooms to discuss!
+</section>
+
+## Section 2: Develop Your Professional Story
+
+When you reflected on your meaningful story, you may have thought about what happened to a character in that story, most likely the main character. And that character probably underwent some change throughout the story. That idea of change is going to be a key component of our own professional stories. 
+
+Taking more lessons from Pixar as well as storytelling theory from Joseph Campbell, most stories (in the western tradition at least) follow the 3-Act Arc. Your story should too!
+
+![Acts of a Story](/assets/images/module3/Week3/ActsInAStory.png)
+
+#### Act I (Set-Up)
+
+* Introduction
+* Inciting incident
+* Turning point
+
+#### Act II (Struggle)
+
+* Obstacles & Crisis
+* Looking for ways to solve the problem
+* Learning what it will take to actually solve problem
+
+#### Act III (Finale)
+
+* Climax
+* Denouement/Resolution
+* Show what’s been learned
+* Protagonist is changed
+
+### Two Examples
+
+#### Example 1:
+"I started my career as a high school special education teacher. Then I went to grad school. Then I became a director at a charter school. Then I became a career coach and now I'm here."
+
+#### Example 2:
+"I always knew I wanted to help people, and that motivation led me to education. I started my career as a classroom teacher, and I loved the opportunities I had to connect with students on an individual level. Eventually, I felt there were limitations in K-12 education, and I wanted to look for something different. After some exploration, I found that I could help people in a new way by combining my love of teaching and connecting one on one with people through professional coaching. I aim to continue helping people through providing them with the tools they need to transition careers and realize their full potential."

```

Comment:
Question: would it be better to include a story that this particular group can relate to? This is an easy story, but it includes going to graduate school, which is kind of the antithesis of going through a program like Turing.